ansible_collections/arista/avd/roles/eos_designs/docs/how-to/avd-to-cvaas.md

```diff
@@ -0,0 +1,147 @@
+# How to integrate AVD with CVaaS
+
+The purpose of this tutorial is to review the steps required in order to integrate AVD with Arista CloudVision-as-a-Service.
+
+## Within CVaaS navigate to
+
+## Steps to create service accounts on CloudVision
+
+1. Go to Settings and Tools --> Access Control --> Service Accounts --> click `+ New Service Account`
+
+```text
+Account name: AVD
+Description: "Automation with AVD"
+Give a description under "Generated Service Account Token"
+Specify the "valid until" date.
+Make sure to copy the generated password. You only get view it once.
+Click "Save" to exit the dialogue box.
+```
+
+![Figure: 1](../../../../media/serviceaccount1.png)
+![Figure: 2](../../../../media/serviceaccount2.png)
+![Figure: 3](../../../../media/serviceaccount3.png)
+
+> NOTE The name of the service account must match a username configured to be authorized on EOS, otherwise device interactive API calls might fail due to authorization denial.
+
+## Add CVAAS to your Ansible inventory file
+
+```text
+DC1:
+  children:
+    CVAAS:
+      hosts:
+        cvaas:
+```
+
+## Adding group variables for the CVaaS instance
+
+Create a folder under `group_vars` named `CVAAS` and add a file named `cvaas_auth.yml`. Your file should look similar to this:
+
+```text
+ansible_host: www.arista.io
+ansible_user: cvaas
```

Comment:
should we add a note here that this must be either `cvaas` or `svc_account` regardless of what your username is, as these two are specially treated to tell the module's backend to use a service account token  as the bearer token rather the legacy user/password based auth in on-prem? I had to add something like this to the cvaas-migration repo as some folks found it confusing: https://github.com/arista-netdevops-community/cvaas-migration#a-note-on-service-accounts

Reply:
+1